<ui version="4.0" >
 <class>LanguagePage</class>
 <widget class="QWidget" name="LanguagePage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="localeSelectionLabel" >
     <property name="text" >
      <string>Language:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="localeComboBox" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="nameLabel" >
     <property name="text" >
      <string>Name:</string>
     </property>
     <property name="buddy" >
      <cstring>identifierNameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="identifierNameLineEdit" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="iconSelectionLabel" >
     <property name="text" >
      <string>Flag:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="iconComboBox" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="keyboardLayoutLabel" >
     <property name="text" >
      <string>Keyboard layout:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QComboBox" name="keyboardLayoutComboBox" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
